### DISCUSSION TITLE
Apply filter to Default Overlay?

### DISCUSSION THREAD

**Discussion by Nuuki9** [meisnate12/Plex-Meta-Manager#1664] [Q&A] [answered]
Is it possible to apply a filter when using one of the Default Overlay Files?

Specifically, I want to add audio overlays (using the "audio_codec" collection), but only where the resolution is 4k. Is that possible, or only if using a vanilla overlay / collection?

**Comment by chazlarson** [comment, marked as answer]
Filtering is not possible with the defaults generally.

**Reply by Nuuki9** [reply]
Fair enough - I figured that was the case but I still get caught out often enough that I wanted to check.
